### ISSUE TITLE
newly created sufolder (with data) in main (root nextcloud) folder is not visible in synchronize settings neither on "cloud" (web broswer) and not syncing. why? 

### ISSUE THREAD

**Issue by RichardPinka** [nextcloud/desktop#3351]
my NC installation on desktop - Windows 10
NC server ver 17 running on Turris Omnia. 
### Expected behaviour
Tell us what should happen:

In my desktop, I create new folder in the already shared Nexcloud (root) folder. I add data into this folder. 
Nextcloud distinguish new folder with data and sync it to server nextlocud . New folder is visible on the list of the settings of the NC desktop settings window . 
I see new folder on the web browser NextCloud storage place. 

### Actual behaviour
Tell us what happens instead

In my desktop, I create new folder in the already shared Nexcloud (root) folder. I add data into this folder. 
Nextcloud is not distinguishing new folder with data and is not syncing it to server nextlocud .
New folder is NOT visible on the list of the settings of the NC desktop settings window .  (You cannot simply "add" it because is already "inside the shared folder")

 I  do not see new folder on the web browser NextCloud storage place. 

### Steps to reproduce
1. I create new folder inside shared root Nextcloud folder in my desktop (Windows 10 64bit)
2. I add data inside it (pictures inside more folder for example)
3. And i expect to have it synced on some time. 

### Client configuration
Client version:  3.1.3 windows on desktop

Operating system: Windows 10 64bit English

OS language: English 

Qt version used by client package (Linux only, see also Settings dialog):

Client package (From Nextcloud or distro) (Linux only):

Installation path of client:

### Server configuration

Nextcloud version: Nextcloud 17.0.5

Storage backend (external storage):

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

1. Client logfile:    download here:   [https://www.uschovna.cz/zasilka/KWEYUW6VUEUGFIV2-HYG/](url)

2. Web server error log:  i do not know where can i find this

3. Server logfile: nextcloud log (data/nextcloud.log): [https://www.uschovna.cz/zasilka/KWC7F5NSB5ZCZRH3-WBH/](url)

**Comment by FlexW** [comment]
The link to your logs does not work.

That I understand you correct: You added a folder A for synchronisation. In this folder A you created images and other files. These files in folder A sync fine? Then you created another folder B inside folder A? This folder B gets not synced?

**Comment by RichardPinka** [comment]
Dear FlexW, thank You for Your feedback and questions. 
lets say, i have folder ROOT which is synchronized with all   dolders/files structure  on both : on my cloud server and on my desktop. 
- i create folder via web browser (in my nextcloud server) in the ROOT folder with name : folderCreatedInServer and i put inside some test data file (e.g. one pdf)
- i create folder on my desktop ROOT folder (this ROOT is synchronized with all structure with ROOT on server) with name: folderCreatedInDesktop and i put inside some test data file (e.g. one pdf)

results:  
folderCreatedInDesktop should be synchronized inside ROOT and should be visible in ROOT placed on the server, which is not. 
folderCreatedInServe should be synchronized inside ROOT and should be visible in ROOT placed on my desktop which is not. 

I realized, i have nextcloud server version Nextcloud 17.0.5  and I need to get updated it, which is bit complicated (i am running it on Turris Omnia ) ,   I will be back after updating this and re-checking wether  such creating new folders in the data strucure will be working . 
thank You for now

**Comment by FlexW** [comment]
@RichardPinka Thanks for the detailed explanation. The if I understand correct, sync is completely not working for you? That should decently worked. Are you sure you configured the sync folders correct? But yes, please upgrade the server version because this version is not supported anymore.

**Comment by RichardPinka** [comment]
@FlexW  "sync is completely not working for me" is not the correct explanation. Data in already synced structure seems to be synchronized . When I insert some pdf document to already existing structure of folders, it will be synced (structure is visible on the shared folders you select on "Nextcloud Settings" window of the desktop client)

Newly created directory (on desktop) has problem to be appeared on the Cloud (i use my "admin" profile on desktop for sync my data). Interestingly, when I created another profile, let say, with name "User1" , newly created directory (in shared folder prepared for such profile "User1") which is my later experience but make into my understanding the issue more confusion. 

Moreover, (maybe important detail to tell):
when desktop client is syncing, in "Nextcloud Settings" window, when You see status of Your sinced folder structure, it is writing info message something like :  12 GB of 185 GB, file 1580 of 92201 .. 
But it often (when it want to check new data and probably sync) starts from zero ... and synchronization looks like lats for ages and forever..  Is it correctly working ? Does it need always to check file by file, from beginning, wether it has been changed ? Nextcloud doesnot reckognize the changes on its list of the synced data and sync "only what it needs" simiralrly, like OneDrive does ? 




I send You logfiles from server  (by the way, by which toll are you evaluating it ?) link here:  [https://www.uschovna.cz/zasilka/MA45C9YPEU4R6I6E-FTZ/]
Log file from desktop client i did unable to create : when i click on settings, app starts "not responding" and freezing.

**Comment by FlexW** [comment]
The client logs are important for this problem. You can find the on the disk in C:/Users/USER/AppData/Roaming/Nextcloud/logs.

> Newly created directory (on desktop) has problem to be appeared on the Cloud (i use my "admin" profile on desktop for sync my data). Interestingly, when I created another profile, let say, with name "User1" , newly created directory (in shared folder prepared for such profile "User1") which is my later experience but make into my understanding the issue more confusion.

Sorry I can not understand that. You have the problems with the user admin, but not with user User1?

> Moreover, (maybe important detail to tell):
when desktop client is syncing, in "Nextcloud Settings" window, when You see status of Your sinced folder structure, it is writing info message something like : 12 GB of 185 GB, file 1580 of 92201 ..
But it often (when it want to check new data and probably sync) starts from zero ... and synchronization looks like lats for ages and forever.. Is it correctly working ? Does it need always to check file by file, from beginning, wether it has been changed ? Nextcloud doesnot reckognize the changes on its list of the synced data and sync "only what it needs" simiralrly, like OneDrive does ?

No, just one the first time it should sync all the files. Did you wait until the sync was finished before you created the directories?

**Comment by RichardPinka** [comment]
> The client logs are important for this problem. You can find the on the disk in

Yes.. see attached logs from my desktop client: [https://www.uschovna.cz/zasilka/MF97YZ6G9F83FAI2-49B/](url)


> Sorry I can not understand that. You have the problems with the user admin, but not with user User1?
Yes, that is correct explanation. I double checked. When I create some new folder in existing structure below e.g. User1 (other user than admin), new folder (with test files) was visible and did sync with cloud, suprisingly. 
But this was not never working with my original "admin" profile, used for myself. 


> No, just one the first time it should sync all the files. Did you wait until the sync was finished before you created the directories?

I did wait .. well better to tell , i saw many times that desktop client has "marker in green circle" which told me, synchronization has been finished.   Anytime i did change some file or structure, the "syncrhonization" process started. (and as far I remember, it started from the beginning,  e.g.  " file 0 of 92201 "    in the info banner of the settings window.  
I am not sure if this normal or wrong procedure, but it seems for me that synchronization is never-ending process. 

actually, cloud server had fall recently, surprisingly , see the comment: 
Internal Server Error

The server encountered an internal error and was unable to complete your request.
Please contact the server administrator if this error reappears multiple times, please include the technical details below in your report.
More details can be found in the server log.

so here is log downloaded from cloud /srv/www/nextlcoud/data location : 
[https://www.uschovna.cz/zasilka/MFF7EKZ4TZ283D7I-UIV/](url)
